packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # @scale-codec/core
 
+## 2.0.0
+
+### Major Changes
+
+- 7e81bbc: Follows the major update of the re-exported `@scale-codec/enum` lib.
```

Comment:
```suggestion
- 7e81bbc: Follow the major update of the re-exported `@scale-codec/enum` lib.
```

I'd also suggest saying which update it was

Reply:
Said